src/components/Basket/Basket.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { getBasket } from '../../store/index';
+import { GoodsList } from './GoodsList';
+
+export const Basket: React.FC = () => {
```

Comment:
Это можно коротко записывать
```suggestion
import React, { FC } from 'react';
import { useSelector } from 'react-redux';
import { getBasket } from '../../store/index';
import { GoodsList } from './GoodsList';

export const Basket: FC = () => {
```